xbmc/music/MusicDatabase.cpp

```diff
@@ -612,6 +622,12 @@ int CMusicDatabase::AddSong(const int idAlbum,
     if (m_pDS->num_rows() == 0)
     {
       m_pDS->close();
+      bool isBoxset = IsAlbumBoxset(idAlbum);
+      if (isBoxset && strDiscSubtitle.empty()) // boxset tag set
+      { // create a dummy title
+        int discno = iTrack >> 16;
+        strDiscSubtitle = StringUtils::Format("%s %i", g_localizeStrings.Get(427), discno);
+      }
```

Comment:
Comments could explain better,  "dummy" sounds like a hack (sorry, I'll be a white space in commments monitor next)
```
    // All songs of a boxset must have a disc subtitle, use "Disc N" when not given in tags
    bool isBoxset = IsAlbumBoxset(idAlbum);
    if (isBoxset && strDiscSubtitle.empty())
    {
      int discno = iTrack >> 16;
      strDiscSubtitle = StringUtils::Format("%s %i", g_localizeStrings.Get(427), discno);
    }
```